common/djangoapps/student/models.py

```diff
@@ -1196,6 +1196,13 @@ def users_enrolled_in(cls, course_id):
         )
 
     @classmethod
+    def enrolled_and_dropped_out_users(cls, course_id):
```

Comment:
This should be called `enrolled_or_dropped_out_users` since it returns users in either the enrolled or unenrolled states.